Casks/s/sanctum.rb

```diff
@@ -0,0 +1,32 @@
+cask "sanctum" do
+  arch arm: "aarch64", intel: "x64"
+  folder = on_arch_conditional arm: "aarch64", intel: "x86_64"
+
+  version "1.7.0"
+  sha256 arm:   "7d0826a6aa7b0e2c502e786eb00a9339b1d48871ad216958b87ac4cd68ee5a6c",
+         intel: "8e78706086b346326441f53206074eb2ab087fb0ef9a67f8c6a154451c14ce97"
+
+  url "https://sanctum.ai/darwin-#{folder}/Sanctum_#{version}_#{arch}.dmg",
+      user_agent: :fake
+  name "Sanctum"
+  desc "Run & interact with full-featured open-source LLMs locally"
+  homepage "https://sanctum.ai/"
+
+  livecheck do
+    url :homepage
+    regex(/Sanctum[._-]?v?(\d+(?:\.\d+)+)[._-]?\w+\.dmg/i)
+    strategy :page_match
```

Comment:
```suggestion
    url :homepage
    regex(/href=.*?Sanctum[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
```

We can omit specifying the `strategy`, as `PageMatch` is used by default here.
When matching file names, we typically restrict matching to `href` attributes and do not make delimiters optional (see [documentation](https://docs.brew.sh/Brew-Livecheck#file-names)).